app.rb

```diff
@@ -23,10 +13,75 @@ def escape(text)
   status 404
 end
 
+class Memo
+  require 'pg'
+  attr_accessor :id, :text, :title
+
+  def self.escape(text)
+    CGI.escape_html(text)
+  end
+
+  def self.unescape(text)
+    CGI.unescape_html(text)
+  end
+
+  def self.all
+    conn = PG.connect(dbname: 'try_sinatra_db')
+    conn.exec('SELECT * FROM memos') do |r|
+      memos = r.map do |row|
+        memo = Memo.new
+        memo.id = row['memo_id']
+        memo.title = unescape(row['title'])
+        memo.text = unescape(row['text'])
+        memo
```

Comment:
`Memo`クラスのコンストラクタを実装して、
```suggestion
        Memo.new(id: row['memo_id'], title: row['title'], text: row['text'])
```
みたいな感じにした方が簡潔かなと思いました。

Reply:
確かにおっしゃる通りです！！🙇‍♂️🙇‍♂️